### PR TITLE
feat: add task view on ProjectDetailsScreen

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#78173

### CODE REVIEW

**Comment by gca-axelor** [comment]
need rebase when https://github.com/axelor/axelor-mobile/pull/547 && #548  is merge

File: packages/apps/project/src/api/project-task-api.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  statusToFilter,
+  priorityToFilter,
+}) => {
+  const criteria = [getSearchCriterias('project_projectTask', searchValue)];
+
+  criteria.push({
+    fieldName: 'id',
+    operator: 'in',
+    value: idsProjectTask,
+  });
```
Comment (vhu-axelor) [inline packages/apps/project/src/api/project-task-api.js:39]: If there is no condition you can directly put in in the array

File: packages/apps/project/src/api/project-task-api.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+  if (userId != null) {
+    criteria.push({
+      fieldName: 'assignedTo.employee.user.id',
```
Comment (vhu-axelor) [inline packages/apps/project/src/api/project-task-api.js:43]: 'assignedTo.id'

File: packages/apps/project/src/components/molecules/TaskActionCard/TaskActionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          name={name}
+          assignedTo={assignedTo}
+          taskDeadline={taskDeadline}
+          parentTask={parentTask}
+          progress={progress}
+          priority={priority}
+          status={status}
+        />
+      </View>
+      <View style={styles.iconContainer}>
```
[the remark this answers is not in the document]
Comment (gca-axelor) [reply packages/apps/project/src/components/molecules/TaskActionCard/TaskActionCard.tsx:62]: if i remove it the card going insane

[the remark this answers is not in the document]
Comment (vhu-axelor) [reply packages/apps/project/src/components/molecules/TaskActionCard/TaskActionCard.tsx:58]: Do it like I told you to do for PartnerActioncard

**Pull request merged**